datadog/fwprovider/resource_datadog_on_call_schedule.go

```diff
@@ -0,0 +1,802 @@
+							Validators:  []validator.String{validators.TimeFormatValidator(time.RFC3339)},
+						},
+						"name": schema.StringAttribute{
+							Required:    true,
+							Description: "The name of this layer.",
+						},
+						"rotation_start": schema.StringAttribute{
+							Optional:    true,
+							Description: "The date/time when the rotation for this layer starts (in ISO 8601).",
+							Validators:  []validator.String{validators.TimeFormatValidator(time.RFC3339)},
+						},
+					},
+					Blocks: map[string]schema.Block{
+						"members": schema.ListNestedBlock{
+							Validators: []validator.List{
+								listvalidator.SizeAtLeast(1),
+							},
+							NestedObject: schema.NestedBlockObject{
+								Blocks: map[string]schema.Block{
+									"user": schema.SingleNestedBlock{
+										Description: "The user assigned to this member. Can be omitted for empty members.",
+										Attributes: map[string]schema.Attribute{
+											"id": schema.StringAttribute{
+												// Member can be empty, so we need it to allow optional
+												Optional:    true,
+												Description: "The user's ID. Can be omitted for empty members.",
+											},
+										},
+									},
+								},
+							},
+						},
+						"restrictions": schema.ListNestedBlock{
+							NestedObject: schema.NestedBlockObject{
+								Attributes: map[string]schema.Attribute{
+									"end_day": schema.StringAttribute{
+										Optional:    true,
+										Validators:  []validator.String{validators.NewEnumValidator[validator.String](datadogV2.NewLayerAttributesRestrictionsItemsEndDayFromValue)},
+										Description: "The weekday when the restriction period ends (Monday through Sunday).",
+									},
+									"end_time": schema.StringAttribute{
+										Optional:    true,
+										Description: "The time of day when the restriction ends (hh:mm:ss).",
+									},
+									"start_day": schema.StringAttribute{
+										Optional:    true,
+										Validators:  []validator.String{validators.NewEnumValidator[validator.String](datadogV2.NewLayerAttributesRestrictionsItemsStartDayFromValue)},
+										Description: "The weekday when the restriction period starts (Monday through Sunday).",
+									},
+									"start_time": schema.StringAttribute{
+										Optional:    true,
+										Description: "The time of day when the restriction begins (hh:mm:ss).",
+									},
+								},
+							},
+						},
+						"interval": schema.SingleNestedBlock{
+							Attributes: map[string]schema.Attribute{
+								"days": schema.Int32Attribute{
+									Optional:    true,
+									Computed:    true,
+									Description: "The number of full days in each rotation period.",
+									Default:     int32default.StaticInt32(int32(0)),
+								},
+								"seconds": schema.Int64Attribute{
+									Optional:    true,
+									Computed:    true,
+									Description: "Extra seconds that may be added to extend the rotation beyond whole days.",
```

Comment:
```suggestion
									Description: "For intervals that are not expressible in whole days, this will be added to `days`.",
```

From the doc it's not immediately clear that interval can also be less than one day